apps/emqx_authz/etc/emqx_authz.conf

```diff
@@ -1,4 +1,18 @@
 emqx_authz:{
+    ## Enable ACL check.
+    ##
+    ## @doc emqx_authz.enable
+    ## ValueType: Boolean
+    ## Default: false
+    enable: true
```

Comment:
Does this look like it sets a global ACL enablement?

So is a configuration like the following still supported?
```
zone.external.enable_acl = true
zone.internal.enable_acl = false
```